doc/eclipse.html

```diff
@@ -301,7 +301,8 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
 <li><b>Output directory</b>: the path where to store the output including temp files, report files, etc. By default, the path is relative to each project except you check the option <b>Absolute ouput path</b> below.</li>
 <li><b>Disalbe default listeners</b>: disable the default listeners when launch testng runtime</li>
 <li><b>Show view when test complete</b>: active the TestNG result view when the test complete</li>
-<li><b>Use Project TestNG jar</b>: It's default to be checked which means TestNG Eclipse Plugin will use the project TestNG jar (for example, the TestNG version managed with maven) when launch TestNG runtime. Otherwise, the TestNG eclipse plugin embedded TestNG version will be used.<br/>
+<li><strike><b>Use Project TestNG jar</b></strike>: (<b>NOTE: this property was removed since 6.9.11, which means TestNG plugin won't append the plugin's embedded testng jar to the runtime test process.</b>) <br/>
+It's default to be checked which means TestNG Eclipse Plugin will use the project TestNG jar (for example, the TestNG version managed with maven) when launch TestNG runtime. Otherwise, the TestNG eclipse plugin embedded TestNG version will be used.<br/>
```

Comment:
I made a few updates, take a look and let me know what you think.

-- 
Cédric


On Mon, Apr 4, 2016 at 4:44 AM, Cédric Beust ♔ <cedric@beust.com> wrote:

> Actually, I'm just going to update the screen shot so it no longer shows
> this checkbox and then remove that part of the documentation.
>
> Ok with you?
>
>
> --
> Cédric
>
>
> On Mon, Apr 4, 2016 at 4:22 AM, Nick Tan <notifications@github.com> wrote:
>
>> In doc/eclipse.html
>> <https://github.com/testng-team/testng-team.github.io/pull/7#discussion_r58320559>
>> :
>>
>> > @@ -301,7 +301,8 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
>> >  <li><b>Output directory</b>: the path where to store the output including temp files, report files, etc. By default, the path is relative to each project except you check the option <b>Absolute ouput path</b> below.</li>
>> >  <li><b>Disalbe default listeners</b>: disable the default listeners when launch testng runtime</li>
>> >  <li><b>Show view when test complete</b>: active the TestNG result view when the test complete</li>
>> > -<li><b>Use Project TestNG jar</b>: It's default to be checked which means TestNG Eclipse Plugin will use the project TestNG jar (for example, the TestNG version managed with maven) when launch TestNG runtime. Otherwise, the TestNG eclipse plugin embedded TestNG version will be used.<br/>
>> > +<li><strike><b>Use Project TestNG jar</b></strike>: (<b>NOTE: this property was removed since 6.9.11, which means TestNG plugin won't append the plugin's embedded testng jar to the runtime test process.</b>) <br/>
>> > +It's default to be checked which means TestNG Eclipse Plugin will use the project TestNG jar (for example, the TestNG version managed with maven) when launch TestNG runtime. Otherwise, the TestNG eclipse plugin embedded TestNG version will be used.<br/>
>>
>> updated.
>> and i removed the 'When to check this option?' section.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/testng-team/testng-team.github.io/pull/7/files/176fea9f0853cd3f4725af0ad51ab7518b39f662#r58320559>
>>
>
>